floodviz/static/js/hydrograph.js

```diff
@@ -228,6 +230,16 @@ var hydromodule = function (options) {
 			.style("font-size", "14px")
 			.text("Discharge (cubic feet per second)");
 
+		var button = d3.select(".btn");
+		button.on('click', function() {reset_hydrograph();});
+
+		if (default_display_ids.toString() === options.display_ids.toString()) {
```

Comment:
This comparision makes me nervous. You are relying on the ids being in the same order. Okay for now, but it seems to me we are doing far too much in this update procedure. It is over 150 lines. If we had more time, I would look at refactoring the hydrograph module.